### PR TITLE
chore: add jina version, dependabot module

### PR DESCRIPTION
**Pull request by deepankarm**
- Added jina version either in `Dockerfile` or `requirements.txt` for all executors (depending on usage). The versions are picked from **GET hubapi.jina.ai/images**.
- Each executor is now listed in `dependabot.yml` file ([Dependabot doesn't support recursive package finding yet](https://github.com/dependabot/dependabot-core/issues/1015))

https://docs.google.com/spreadsheets/d/1o7G9mDNjHy9j6lfNs7NayeX-XYKbH42cOcElduV3LAo/edit#gid=0

### CODE REVIEW

File: crafters/audio/AudioMonophoner/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM jinaai/jina
+FROM jinaai/jina:1.0.15
```
Comment (nan-wang) [inline crafters/audio/AudioMonophoner/Dockerfile:1]: Will the jina version be updated automatically when the update action is triggered?

File: classifiers/nlp/ZeroShotTFClassifier/requirements.txt [hunk 2]
```diff
@@ -1,2 +1,3 @@
+jina==1.0.15
```
Comment (nan-wang) [inline classifiers/nlp/ZeroShotTFClassifier/requirements.txt:1]: Do we always need to specify the jina version in the requirements? `jina build` will automatically add the version information.

```suggestion
jina==1.0.15
```

**Review by nan-wang — CHANGES_REQUESTED** [review]

File: crafters/audio/AudioMonophoner/Dockerfile (hunk 1 above)
Comment (deepankarm) [reply crafters/audio/AudioMonophoner/Dockerfile:1]: Yes, Example PR: https://github.com/jina-ai/jina-hub/pull/7096/

File: classifiers/nlp/ZeroShotTFClassifier/requirements.txt (hunk 2 above)
Comment (deepankarm) [reply classifiers/nlp/ZeroShotTFClassifier/requirements.txt:1]: To enable version bumps, jina version must be mentioned either in `Dockerfile` or `requirements.txt`. If we don't pass a version number, there won't be any "version bump" PRs. As we are reworking on cookiecutter, during executor creation, we can fill in the version number too.

**Review by nan-wang — APPROVED** [review]
LGTM👍

**Pull request merged**